packages/angular/projects/angular-sdk/src/lib/feature-flag.directive.ts

```diff
@@ -69,14 +70,17 @@ export abstract class FeatureFlagDirective<T extends FlagValue> implements OnDes
     templateRef: TemplateRef<FeatureFlagDirectiveContext<T>>,
   ) {
     this._thenTemplateRef = templateRef;
-    this.initClient();
   }
 
   set featureFlagDomain(domain: string | undefined) {
     this._featureFlagDomain = domain;
     this.initClient();
```

Comment:
Can this be removed now?

Reply:
No it sadly can not. 
If we remove this, angular will not see the changes in the domain and so it will not use the new one.

Reply:
I'm not sure I follow. Perhaps it's worth adding a comment so that future us won't accidentally remove it 😄

Reply:
Added a comment @beeme1mr.